core/src/mender-client.c

```diff
@@ -1213,13 +1213,13 @@ mender_client_update_work_function(void) {
                 /* fallthrough */
 
             case MENDER_UPDATE_STATE_COMMIT:
+                printf("c MENDER_UPDATE_STATE_COMMIT\n");
```

Comment:
Is this leftovers from a debugging session?

Reply:
Touché :person_fencing: